lib/models/movie.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
+    return {
+      id: this.get('id'),
+      title: this.get('title'),
+      release_year: this.get('release_year'),
```

Comment:
Are these db lookups?

Reply:
i think serialize takes in an instance of a movie and then sets a db entry with the corresponding values

Reply:
Thx